caitlinSolution.js

```diff
@@ -0,0 +1,88 @@
+var game = {
+  letters: [],
+  guesses: [],
+  lastGuess: "",
+  display: "",
+  wrongGuesses: 0,
+  guessAccuracy: false,
+  rightGuesses: 0,
+  gameInProgress: true,
+  word: "",
+ // choose word -- make this into a prompt soon
+  enterWord: function() {
+    this.word = prompt("What word would you like to play?");
+
+    this.splitWord();
+  },
+
+  // Split word into an array
+  splitWord: function() {
+    this.letters = this.word.split("");
+
+    this.guessLetter();
+  },
+  // Take letter input & store input in guess array
+  guessLetter: function() {
+    this.guessAccuracy = false;
+
+    while (this.gameInProgress === true) {
```

Comment:
Instead of `while (this.gameInProgress === true)`, you can actually just write `while(this.gameInProgress)`, it's saying:
```
while(true) { 
do something
}
```